modin/engines/base/io/text/csv_dispatcher.py

```diff
@@ -220,13 +202,16 @@ def _read_csv_check_support(
         if skiprows is not None and not isinstance(skiprows, int):
             return False
 
+        # can't pickle dialect object to the worker process
+        if read_csv_kwargs.get("dialect", None) is not None:
+            return False
```

Comment:
are you sure this is relevant to the issue you're stating as being closed by this PR?

Reply:
Not sure) change was reverted.